src/Dfe.PlanTech.Domain/Persistence/Models/CacheRefreshConfiguration.cs

```diff
@@ -1,3 +1,9 @@
 namespace Dfe.PlanTech.Domain.Persistence.Models;
 
-public record CacheRefreshConfiguration(string? Endpoint, string? ApiKeyName, string? ApiKeyValue);
+public record CacheRefreshConfiguration(string? Endpoint, string? ApiKeyName, string? ApiKeyValue)
+{
+  public CacheRefreshConfiguration() : this(null, null, null)
+  {
```

Comment:
why did you add this?, is it for defaults? / (doesn't work on line 3?)

Reply:
1. I added the [options pattern](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-8.0) to this and the `ContentfulOptions` setup. This was actually not necessary but simplified up the classes.
2. The `.Bind()` method requires a parameterless constructor for the target class; so I had to add parameterless constructors
3. I thought the easiest way to do this was the way I've done it. But to add another constructor, with a default constructor, requires that you call the default constructor. 

Sooo... Yeah. For defaults.